src/components/Send.jsx

```diff
@@ -640,7 +640,11 @@ export default function Send() {
                         {settings.useAdvancedWalletMode
                           ? t('send.account_selector_option_dev_mode', { number: account })
                           : t('send.account_selector_option', { number: account })}{' '}
-                        {settings.showBalance && `(\u20BF${balance})`}
+                        {settings.showBalance
+                          ? settings.unit === 'sats'
+                            ? `(${formatSats(balance * 100000000)} sats)`
```

Comment:
We have a `btcToSats` function in `src/utils.ts`. It's already imported in this component, so you should be able to just do:

```suggestion
                            ? `(${formatSats(btcToSats(balance))} sats)`
```